Elaborated on storing of files (#5038)
umbraco/UmbracoDocs: 11/umbraco-forms/editor/attaching-workflows/workflow-types.md

* File extension (required)
* XsltFile - used to transform the XML

The path needs to point to a folder, not a file name. The files are then stored locally, and relative paths are resolved to the content root.

{% hint style="info" %}

> If you store the files within the wwwroot or App_Plugins folders, this will make the files publicly available by default.

Reviewer: ```suggestion
When storing the files within the `wwwroot` or `App_Plugins` folders, the files will be publicly available by default.
```

Author: Looks good :)
